examples/crm/vite.config.ts

```diff
@@ -46,7 +46,7 @@ export default defineConfig({
         preserveSymlinks: true,
         alias: [
             // allow profiling in production
-            { find: 'react-dom', replacement: 'react-dom/profiling' },
+            // { find: 'react-dom', replacement: 'react-dom/profiling' },
```

Comment:
commented out code

Reply:
Yes, haven't yet investigated how we can add profiling back. I'll address this issue in another PR